### DISCUSSION TITLE
Table pagination also on top of table

### DISCUSSION THREAD

**Discussion by jcerdan** [livewire/flux#350] [Feature Requests] [answered]
Hello,

sometimes if pagination is with 50 records per page it would be useful to also have pagination on top of table. Could you add an option to display also pagination on top?

Best regards

**Comment by joshhanley** [comment, marked as answer]
@jcerdan we've just documented the pagination component which will allow you to add it above your table if you like. See details here https://fluxui.dev/components/pagination
